src/metabase/notification/condition.clj

```diff
@@ -21,18 +24,26 @@
         :<= (apply <= (map #(evaluate-expression % context) operands))
 
         ;; Data access
-        :context (get-in context (map keyword operands))
+        :context (let [v (get-in context (map keyword operands))]
+                   (if (keyword? v)
+                     (u/qualified-name v)
+                     v))
```

Comment:
Maybe this is too powerful, but you could also delegate this to `evaluate-expression`?

Reply:
Hmm nah that probably wouldn't make sense for most lists 😆